orm/internal/testpb/test_schema.cosmos_orm.go

```diff
@@ -10,7 +10,7 @@ import (
 	ormerrors "github.com/cosmos/cosmos-sdk/orm/types/ormerrors"
 )
 
-type ExampleTableStore interface {
+type ExampleTableTable interface {
```

Comment:
Can we have a better name for this? maybe just `ExampleTable` or `ExampleTestTable` ?